docusaurus/docs/iOS/guides/working-with-the-composer.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Working with the Composer
+---
+
+The `ComposerVC` is the component responsible to create new messages or change existing messages. The composer belongs to the `ChatVC` component that integrates the message list and the composer in one view. But if you are not using our `ChatVC` you can use the composer component in isolation with your own chat view as well. The `ComposerVC` manages all the logic and actions of the `ComposerView`, where the latter is only responsible for the composer's layout.
```

Comment:
Link to ComposerVC reference docs (and other components like ChatVC) would be nice in intro

Reply:
How do we reference this? Manually? Would be nice to have a way that even if we change the structure of the docs, the refs are not broken 🤔 But currently this is manual right?

Reply:
```suggestion
The [`ComposerVC`](../ReferenceDocs/Sources/StreamChatUI/Composer/ComposerVC.md) is the component responsible to create new messages or change existing messages. The composer belongs to the `ChatVC` component that integrates the message list and the composer in one view. But if you are not using our `ChatVC` you can use the composer component in isolation with your own chat view as well. The `ComposerVC` manages all the logic and actions of the `ComposerView`, where the latter is only responsible for the composer's layout.
```


@nuno-vieira just take a look at other guides, the structure doesn't matter because you reference to the actual location of the file instead of some docusaurus alias... see `Working with messagelist layout` and the suggestion :)